docs/src/main/asciidoc/extension-codestart.adoc

```diff
@@ -91,31 +91,42 @@ The codestart should not include any business logic, instead, it should contain
 
 - Optionally, add the <<app-config>>.
 
-- Create an <<integration-test>> *You can use it to get your codestart generated as a project*.
+- Create an <<integration-test>>.
 
-You can also use it for dev using the `buildAllProjectsForLocalUse` (Disabled by default because it's already covered by `QuarkusCodestartBuildIT`).
+- <<generating>>
 
-If you want to try using the CLI or Maven plugin for generating a project with your codestart:
+== Writing an Extension Codestart in the Quarkiverse or standalone
+
+For extensions hosted outside of the Quarkus core[https://github.com/quarkusio/quarkus] repository, codestarts will typically be located in the runtime module (with special instruction in the `pom.xml` to generate a separate codestart artifact). https://github.com/ia3andy/aloha-code/[Here, window="_blank"] is an example extension with a codestart & its tests
+
+
+[#generating]
+== Generating your Extension Codestart
```

Comment:
This part is specific to codestarts in the Quarkus repo, right? Should we mention that explicitly?